src/libm/sleefsimdsp.c

```diff
@@ -908,6 +911,9 @@ EXPORT CONST VECTOR_CC vfloat xtanf(vfloat d) {
 
   q = vrint_vi2_vf(vmul_vf_vf_vf(d, vcast_vf_f((float)(2 * M_1_PI))));
   u = vcast_vf_vi2(q);
+#if (defined(ENABLE_PUREC_SCALAR) || defined(ENABLE_PURECFMA_SCALAR) || defined(ENABLE_VXE) || defined(ENABLE_VXENOFMA) || defined(ENABLE_VXE2) || defined(ENABLE_VXE2NOFMA)) && !defined(__clang__) && __GNUC__ == 13
```

Comment:
Is it just a problem with gcc 13?
If so we should seek to understand what is going on, because that might have repercussions elsewhere that we haven't noticed.

Reply:
If you wish to merge I would recommend creating an issue, so we don't forget about it.

Reply:
There seems no problem with gcc-12 and gcc-14.
Of course it is better to understand what is going on, but that takes quite a bit of time.
I will create an issue for this matter.